Makefile

```diff
@@ -1,13 +1,11 @@
-VIRTUALENV = virtualenv-2.7
+VIRTUALENV = virtualenv
```

Comment:
we're forcing 2.7 for this project, and virtualenv is not 2.7 by default on many platforms. so let's keep -2.7